src/main/internal-app-schemes.ts

```diff
@@ -88,23 +87,13 @@ app.once('ready', () => {
     }
 
     const otherFolder = ALLOWED_FOLDERS.findIndex(folder => {
-      console.log(
-        process.platform,
-        file,
-        folder + '\\',
-        file.startsWith(folder + '\\')
-      )
       return process.platform === 'win32'
         ? file.startsWith(folder + '\\')
         : file.startsWith(folder + '/')
```

Comment:
why not use `path.sep` https://nodejs.org/api/all.html#path_path_sep?

Reply:
Oooh good idea :)